boards/toradex/verdin_imx8mm/doc/index.rst

```diff
@@ -0,0 +1,268 @@
+| UART_2        | UART3           | General purpose UART      |
++---------------+-----------------+---------------------------+
+| UART_3        | UART1           | Cortex-A53 debug UART     |
++---------------+-----------------+---------------------------+
+| UART_4        | UART4           | Cortex-M4 debug UART      |
++---------------+-----------------+---------------------------+
+
+GPIO:
+
+All the GPIO banks are enabled in the :zephyr_file:`dts/arm/nxp/nxp_imx8m_m4.dtsi`.
+
+LED:
+
+There is no LED in the module itself, this is dependent on the carrier board that
+is being used with the module. The device tree is configured to use the GPIO3_IO1,
+which can be connected to the LED of the Verdin Development Board or changed in the
+:zephyr_file:`boards/arm/verdin_imx8mm_m4.dts` if needed.
+
+System Clock
+============
+
+The M4 Core is configured to run at a 400 MHz clock speed.
+
+Programming and Debugging
+*************************
+
+The i.MX8MM doesn't have QSPI flash for the M4 and it needs
+to be started by the A53 core. The A53 core is responsible to load the M4 binary
+application into the RAM, putting the M4 in reset, setting the M4 Program Counter and
+Stack Pointer, and get the M4 out of reset. The A53 can perform these steps at the
+bootloader level or after the Linux system has booted via RemoteProc.
+
+The M4 can use up to 3 different RAMs. These are the memory mapping for A53 and M4:
+
++------------+-------------------------+------------------------+-----------------------+----------------------+
+| Region     | Cortex-A53              | Cortex-M4 (System Bus) | Cortex-M4 (Code Bus)  | Size                 |
++============+=========================+========================+=======================+======================+
+| OCRAM      | 0x00900000-0x0093FFFF   | 0x20200000-0x2023FFFF  | 0x00900000-0x0093FFFF | 256KB                |
++------------+-------------------------+------------------------+-----------------------+----------------------+
+| TCMU       | 0x00800000-0x0081FFFF   | 0x20000000-0x2001FFFF  |                       | 128KB                |
++------------+-------------------------+------------------------+-----------------------+----------------------+
+| TCML       | 0x007E0000-0x007FFFFF   |                        | 0x1FFE0000-0x1FFFFFFF | 128KB                |
++------------+-------------------------+------------------------+-----------------------+----------------------+
+| OCRAM_S    | 0x00180000-0x00187FFF   | 0x20180000-0x20187FFF  | 0x00180000-0x00187FFF | 32KB                 |
++------------+-------------------------+------------------------+-----------------------+----------------------+
+
+For more information about memory mapping see the
+`i.MX 8M Applications Processor Reference Manual`_  (section 2.1.2 and 2.1.3)
+
+At compilation time you have to choose which RAM will be used. This
+configuration is done in the file ``boards/arm/verdin_imx8mm_m4/verdin_imx8mm_m4.dts``
```

Comment:
File path out of date, also please use `zephyr_file` here

Reply:
This has been addressed in the latest push.